docs/reference/indices/open-close.asciidoc

```diff
@@ -32,3 +32,10 @@ This setting can also be changed via the cluster update settings api.
 
 Closed indices consume a significant amount of disk-space which can cause problems in managed environments. Closing indices can be disabled via the cluster settings
 API by setting `cluster.indices.close.enable` to `false`. The default is `true`.
+
+Most APIs, including _stats and _search, return `index_closed_exception` and 403 status code when accessing closed index.  The `ignore_unavailable` querystring parameter allows to control whether closed index cause an error to be returned or not.  Call to index `_aliases` returns correct value without error. 
```

Comment:
I find this sentence confusing: "Call to index `_aliases` returns correct value without error. " I would remove it.